db/migrate/20150602114114_add_hidden_flag_to_lookup_values.rb

```diff
@@ -0,0 +1,5 @@
+class AddHiddenFlagToLookupValues < ActiveRecord::Migration
+  def change
+    add_column :lookup_values, :hidden, :boolean, :default => false
```

Comment:
Shouldn't this be on the lookup key, not the value?  I don't think you'd want the risk of missing it on one override when it could simply be set for the parameter as a whole.